src/pathfora.js

```diff
@@ -1750,23 +1749,32 @@
         }
       };
 
-      var contentCb = function (content, w) {
-        if (content) {
-          w.content = {
-            0: {
-              title: content.title,
-              description: content.description,
-              url: 'http://' + content.url,
-              image: content.primary_image
+      var recContent = function (w) {
+        pathfora.addCallback(function () {
+          if (pathfora.acctid === '') {
+            pathfora.acctid = context.lio.account.id;
```

Comment:
probably need to make sure that lio and account both exist before assigning. it should be it'll panic if they dont and since we want this to work without lytics, in theory, we should cover all bases.